library/src/scala/NamedTuple.scala

```diff
@@ -128,6 +128,10 @@ object NamedTuple:
   /** The empty named tuple */
   val Empty: Empty = EmptyTuple
 
+  /** The ordering instance for named tuples */
+  given namedTupleOrdering: [N <: Tuple, V <: Tuple] => (ord: Ordering[V]) => Ordering[NamedTuple[N, V]]:
```

Comment:
if you are in the scope of the opaque type why can't you return `ord`?

Reply:
I don't think I understand. How does that relate to the code here?

Reply:
why not this:
```suggestion
  given namedTupleOrdering: [N <: Tuple, V <: Tuple] => (ord: Ordering[V]) => Ordering[NamedTuple[N, V]] = ord
```

Reply:
OK, let's try that!